### PR TITLE
Launch files

### PR DESCRIPTION
**Pull request by saxtot**
New bringup hierarchy for total launch of markhor functions through markhor_top_level.launch.
Naming convention was also revised, some were created before certain launch files.
Multiple files will have to be renamed in accordance.

Modifications according to diagram below. Additionally, includes web_ui and excludes gazebo.
image

### CODE REVIEW

**Review by CloeD — APPROVED** [review]
Seems good to me, but wait for Ludo's approval before merging.

**Comment by lvanasse** [comment]
I've merged this branch into the flippers_control_ros branch, so there's no more different between the two branch. I'll close this PR since there was some modification that I did before inside the flipper_control_ros branch (sorry I should have told you inside the PR, but I was short on time)

**Pull request closed without merging**
